logging/processors.rst

```diff
@@ -46,6 +46,11 @@ using a processor::
         }
     }
 
+.. note::
+
+    As every log record is process by all registred processors, you must be very careful when you create one processor.
+    It has to be an optimized process, to not introduce any overhead.
```

Comment:
We always “challenge” notes like this :). We’ve found that, over the years, more and more edge cause notes tend to be added, which started to make articles less and less readable. 

Making it obvious (ideally via code blocks, but that’s less important) that a processor will be called once for *every* log record is probably a nice thing to mention. Going further to warn that you need to keep them “light” I think adds a bit too much.

My ideal situation would be to remove this note and, if needed, add something to a code block in this section (even a comment in code) that a processor is called once per log entry.

Reply:
Understood, will add a code comment in a processor mentionning this idea